website/docs/docs/workshops/ci-cd/workshop/06_verifying_feature_pacts.md

```diff
@@ -5,65 +5,75 @@ title: Strategies for verifying feature pacts
 
     ```js
     consumerVersionSelectors: [
-        { tag: 'master', latest: true },
-        { tag: 'feat/new-field', latest: true },
+        { mainBranch: true },
+        { branch: 'feat/new-field' },
         ...
     ]
     ```
 
     * This does the job, but it's a bit fiddly, and there are better options.
 
-1. Make a matching feature branch in the provider, and dynamically fetch the pact for the matching consumer branch if it exists (no error will be raised if a pact does not exist for a particular tag). This is a reasonably common approach, where the two teams coordinate feature development using matching branch names.
+2. Make a matching feature branch in the provider, and dynamically fetch the pact for the matching consumer branch if it exists (no error will be raised if a pact does not exist for a particular branch). This is a reasonably common approach, where the two teams coordinate feature development using matching branch names.
 
     ```js
     consumerVersionSelectors: [
-        { tag: 'master', latest: true },
-        { tag: process.env.GIT_BRANCH, latest: true },
+        { mainBranch: true },
+        { branch: process.env.GIT_BRANCH },
         ...
     ]
     ```
 
-1. Enable "work in progress" pacts.
+3. Enable "work in progress" pacts - which is the recommended and option we will be proceeding with
 
 ## Work In Progress Pacts
 
 The "work in progress pacts" feature solves the problem of provider teams having to manually update and commit changes to their verification configuration to publish verifications for feature pacts from CI.
 
-When enabled, it will cause the provider to automatically verify, in pending mode, any "outstanding" pacts (ie. ones for which it has not yet published a successful result), as well as the pacts that were explicitly specified in the tag/selector list.
+When enabled, it will cause the provider to automatically verify, in pending mode, any "outstanding" pacts (ie. ones for which it has not yet published a successful result), as well as the pacts that were explicitly specified in the consumer version selector list (such as a specified `branch`).
 
-A "work in progress" pact is a pact that is the latest for its tag that does not have any successful verification results (ie. is still pending) for the configured provider tag . At this stage in the exercise, the `feat/new-field` pact is still in pending state for the `master` branch of the provider, so it is considered a "work in progress" pact for master.
+A "work in progress" pact is a pact that is the latest for its branch that does not have any successful verification results (ie. is still pending) for the configured provider branch . At this stage in the exercise, the `feat/new-field` pact is still in pending state for the main branch of the provider, so it is considered a "work in progress" pact for `master`.
```

Comment:
It's slightly more complicated than this now. My brain is not working so I'll come back to this.

Reply:
Now I think about it, it's not worth confusing people with the full technical details of how WIP pacts are selected at this stage. What you've got is 👍🏽